autosklearn/ensembles/ensemble_selection.py

```diff
@@ -112,12 +126,19 @@ def _fast(self, predictions, labels):
                     (1. / float(s + 1)) * pred,
                     out=fant_ensemble_prediction
                 )
-                scores[j] = self.metric._optimum - calculate_score(
+                this_score = calculate_score(
                     solution=labels,
                     prediction=fant_ensemble_prediction,
                     task_type=self.task_type,
                     metric=self.metric,
                     all_scoring_functions=False)
+                if isinstance(this_score, float):
```

Comment:
This will most likely become a performance bottleneck. As we know that this won't be a dictionary, can you do a `typing.cast` to turn the type into a float?